public/js/actions/sources.js

```diff
@@ -52,6 +52,30 @@ function newSource(source) {
   };
 }
 
+function loadSourceMap(generatedSource) {
+  return ({ dispatch, getState }) => {
+    let sourceMap = getSourceMap(getState(), generatedSource.id);
+    if (sourceMap) {
+      return {};
+    }
+
+    return dispatch({
+      type: constants.ADD_SOURCE_MAP,
```

Comment:
`LOAD_SOURCE_MAP` might be a better description of this, analogous to `LOAD_SOURCE_TEXT`.